client/state/posts/selectors.js

```diff
@@ -359,6 +363,31 @@ export const isEditedPostDirty = createSelector(
 );
 
 /**
+  * Returns true if the edited post has content
+  * (title, excerpt or content not empty)
+  *
+  * @param  {Object}  state  Global state tree
+  * @param  {Number}  siteId Site ID
+  * @param  {Number}  postId Post ID
+  * @return {Boolean}        Whether the edited post has content or not
+ */
+export function editedPostHasContent( state, siteId, postId ) {
+	const editedPost = getEditedPost( state, siteId, postId );
+	return (
+		!! editedPost &&
+		(
+			some( [ 'title', 'excerpt' ], ( field ) => editedPost[ field ] && !! editedPost[ field ].trim() ) ||
```

Comment:
No need to change, but I like how Lodash methods handle the falsey case for you gracefully. In this case it could be simplified to:

```js
some( [ 'title', 'expect' ], ( field ) => trim( editedPost[ field ] ) )
```